### PR TITLE
=clu #16897 Correct wording for routees.paths

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Comment by patriknw** [comment]
Refs #16897

File: akka-docs/rst/java/cluster-usage.rst [hunk 1]
```diff
@@ -454,7 +454,8 @@ That is not done by the router. The configuration for a group looks like this:
   The routee actors should be started as early as possible when starting the actor system, because
   the router will try to use them as soon as the member status is changed to 'Up'.
 
-The relative actor paths defined in ``routees.paths`` are used as for selecting the actors to which the messages will be forwarded to by the router.
+The actor paths without address information are defined in ``routees.paths`` are used as for selecting the
```
Comment (ktoso) [inline akka-docs/rst/java/cluster-usage.rst]: additional are and as:

the actor paths without address information ~~are~~ defined in ... are used ~~as~~ for

**Comment by ktoso** [comment]
LGTM after typos  fixed

File: akka-docs/rst/java/cluster-usage.rst (hunk 1 above)
Comment (patriknw) [inline akka-docs/rst/java/cluster-usage.rst]: thx

**Pull request merged**